webapp/portlet/src/main/webapp/vue-apps/common/components/ExoSpaceAvatar.vue

```diff
@@ -32,7 +32,7 @@
             class="pull-left text-truncate ms-2">
             <p
               class="text-truncate subtitle-2 my-auto hidden-space">
-              {{ $t('spacesList.label.hiddenSpace') }}
+              {{ displayName }}
```

Comment:
Since, this is a new PR for this old task, I add a remark on it.  The space name shouldn't be accessible when it's hidden and the user isn't member of, please check the property `notAccessibleSpace`. Please check with @AzmiTouil to get more information.